Can you please add a sign off to the commit 
```
Run echo "Checking DCO sign-off for commits in this PR..."
Checking DCO sign-off for commits in this PR...
Error: Commit b37a3ad609eadc1adf698b92068a1c9216154d98 is missing DCO sign-off (Signed-off-by).
  Subject: fix: correct MISSING import and support MOTION_FILE env var in tracking config
  Author:  ruying.bupt <parrot18@users.noreply.github.com>

One or more commits are missing 'Signed-off-by'. Please amend with:
  git commit --amend -s
See CONTRIBUTING.md for details.
Error: Process completed with exit code 1.
```

--- a/agile/rl_env/tasks/tracking/g1/flat_env_cfg.py
+++ b/agile/rl_env/tasks/tracking/g1/flat_env_cfg.py
@@ -13,7 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import MISSING
+import os
+from dataclasses import MISSING
 
 from isaaclab.utils import configclass
 
@@ -30,7 +31,7 @@ class G1FlatEnvCfg(TrackingEnvCfg):
         self.actions.joint_pos.scale = G1_29DOF_ACTION_SCALE
         self.commands.motion.anchor_body_name = "torso_link"
         self.commands.motion.debug_vis = False
-        self.commands.motion.motion_file = MISSING
+        self.commands.motion.motion_file = os.environ.get("MOTION_FILE", MISSING)
         self.commands.motion.body_names = [
             "pelvis",
             "left_hip_roll_link",

